warp/Cargo.toml

```diff
@@ -58,17 +58,19 @@ paste.workspace = true
 tracing = { workspace = true }
 mediatype.workspace = true
 
+wasm-bindgen.workspace = true
+gloo.workspace = true
+web-sys.workspace = true
+js-sys.workspace = true
+console_error_panic_hook.workspace = true
+wasm-streams.workspace = true
+wasm-bindgen-futures.workspace = true
```

Comment:
nit: Lets leave this under `[target.'cfg(target_arch = "wasm32")'.dependencies]` to minimize the dependencies at compile-time.

Reply:
done